service/deployer/eventer/spec.go

```diff
@@ -12,11 +8,11 @@ type DeploymentEvent struct {
 
 // Eventer represents a Service that checks for deployment events.
 type Eventer interface {
-	// NewDeploymentEvents returns a list of DeploymentEvents that have been
-	// created since the previous check, and a minimum time for the controller
-	// to wait before checking again.
-	// In case of error, the error will be non-nil.
-	NewDeploymentEvents() ([]DeploymentEvent, time.Duration, error)
+	// NewDeploymentEvents returns a channel of DeploymentEvents.
+	// This channel can be ranged over to receive DeploymentEvents as they come
+	// in.
+	// In case of error during setup, the error will be non-nil.
+	NewDeploymentEvents() (chan DeploymentEvent, error)
```

Comment:
I thought it'll be `chan []DeploymentEvent` but I like more `chan DeploymentEvent` - inner loop removed!
But is should be receive only channel.